src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java

```diff
@@ -343,26 +343,37 @@ public int getProtocolVersion() {
     @Override
     public String dumpVersionInfo(String buffer) {
         buffer += logAndAddToPasteBinBuffer("Multiverse-NetherPortals Version: " + this.getDescription().getVersion());
-        buffer += logAndAddToPasteBinBuffer("Bukkit Version: " + this.getServer().getVersion());
-        buffer += logAndAddToPasteBinBuffer("World links: " + this.getWorldLinks());
-        buffer += logAndAddToPasteBinBuffer("Nether Prefix: " + netherPrefix);
-        buffer += logAndAddToPasteBinBuffer("Nether Suffix: " + netherSuffix);
+        buffer += logAndAddToPasteBinBuffer("Nether Prefix: " + this.getNetherPrefix());
+        buffer += logAndAddToPasteBinBuffer("Nether Suffix: " + this.getNetherSuffix());
+        buffer += logAndAddToPasteBinBuffer("End Prefix: " + this.getEndPrefix());
+        buffer += logAndAddToPasteBinBuffer("End Suffix: " + this.getEndSuffix());
+        buffer += logAndAddToPasteBinBuffer("Nether Links: " + this.getWorldLinks());
+        buffer += logAndAddToPasteBinBuffer("End Links: " + this.getEndWorldLinks());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Bounceback: " + this.isUsingBounceBack());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Teleport Entities: " + this.isTeleportingEntities());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send Disabled Portal Message: " + this.isSendingDisabledPortalMessage());
+        buffer += logAndAddToPasteBinBuffer("[Multiverse-NetherPortals] Send No Destination Message: " + this.isSendingNoDestinationMessage());
```

Comment:
why are some prefixed and some not?

Reply:
I think the prefix here is wrong cause the `logAndAddToPasteBinBuffer` does that.

Reply:
My mistake. The method isn't used though, it's deprecated and I think we should remove it in MV 5.0.0. Do you still want me to fix it?

Reply:
Fixed it.